Dockerfile

```diff
@@ -706,7 +708,7 @@ ONBUILD RUN find /rootfs \
     && rm -rf /initramfs
 ONBUILD WORKDIR /
 
-FROM installer AS imager
```

Comment:
the `ONBUILD` instructions in `installer` were executed here, while it shouldn't have been